### DISCUSSION TITLE
Browser navigation for assets folder structure

### DISCUSSION THREAD

**Discussion by robinbeatty** [craftcms/cms#14519] [Ideas] [answered]
Some projects we have have assets in rather deep folder structure. Currently when navigating the folders the browser back/forward buttons navigate away from the assets cp page (loosing our place in the folder nav) – it would be useful if we could navigate to previous view instead of previous page. Some tweaks to vue router etc in assets nav would be great!

**Comment by jwb009** [comment, marked as answer]
This has been brought up before, see #12826. I agree it's a needed change...the assets folder should behave the same as Finder or Windows Explorer. It's just intuitive and second nature to expect the back button to go back to the previous view.
